tests/TestRvsdgs.hpp

```diff
@@ -18,9 +18,7 @@ class RvsdgTest {
 	jlm::llvm::RvsdgModule &
```

Comment:
Let's fix the indentation for the entire function, please.

Reply:
I changed all tabs to spaces in the RvsdgTest class, but left the rest of the file alone. Is that what you meant? I hadn't noticed before that this file contains both tabs and spaces, but it seems recent code prefers spaces?

Reply:
Yes, we switched from tabs to spaces at some point. Now, we have a happy mix. I think it is time to get a clang format going and enforce such behavior at some point.